src/main/webapp/success.xhtml

```diff
@@ -4,7 +4,7 @@
       xmlns:f="http://java.sun.com/jsf/core"
       xmlns:p="http://primefaces.org/ui">
 <h:head>
```

Comment:
All .xhtml files should use lowercase 'html' in the type declaration; <!DOCTYPE html>